docs/src/guide/parallelization.md

```diff
@@ -52,60 +52,109 @@ as a breakdown over individual routines.
     For this to take effect recompiling all DFTK (including the precompile cache)
     is needed.
 
-
 !!! note "Timing measurements and threading"
     Unfortunately measuring timings in `TimerOutputs` is not yet thread-safe.
     Therefore taking timings of threaded parts of the code will be disabled
     unless you set `DFTK_TIMING` to `"all"`. In this case you must not use
     Julia threading (see section below) or otherwise undefined behaviour results.
 
-## Threading
-At the moment DFTK employs shared-memory parallelism
-using multiple levels of threading
-which distribute the workload
-over different ``k``-Points, bands or within
-an FFT or BLAS call between processors.
-At its current stage our approach to threading is quite
-simple and pragmatic,
-such that we do not yet achieve a great scaling.
-This should be improved in future versions of the code.
-
-Finding a good sweet spot between the number of threads to use
-and the extra performance gained by each additional working core
-is not always easy,
-since starting, terminating and synchronising threads takes time as well.
-Most importantly the best settings wrt. threading
-depend on both hardware and the problem
-(e.g. number of bands, ``k``-Points, FFT grid size).
-
-For the moment DFTK does not offer an automated selection mechanism
-of thread and parallelization threading and just uses the Julia defaults.
-Since these are rarely good,
-users are advised to use the timing capabilities of DFTK
-to experiment with threading
-for their particular use case before running larger calculations.
-
-### FFTW threads
-For typical small to medium-size calculations in DFTK the largest part of time is spent
-doing discrete Fourier transforms (about 80 to 90%).
-For this reason parallelising FFTs can have a large effect on the runtime
-of larger calculation in DFTK.
-Unfortunately for scaling of FFT threading for smaller problem sizes
-and large numbers of threads is not great,
-such that by default threading in FFTW is even disabled.
-
-The **recommended setting** for FFT threading with DFTK
-is therefore to only use moderate number of FFT threads,
-something like ``2`` or ``4`` and for smaller calculations
-disable FFT threading completely.
-To **enable parallelization of FFTs** (which is by default disabled),
-use
+## Options for parallelisation
+At the moment DFTK offers two ways to parallelise a calculation,
+firstly shared-memory parallelism using threading
+and secondly (possibly distributed) multiprocessing using MPI
```

Comment:
It's always distributed between processes, whether it's also distributed across machines is, from the conceptual point of view, irrelevant